src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -97,8 +77,8 @@ object JesBackend {
     gcsPath.map(JesInput(ExtraConfigParamName, _, Paths.get(""), "LITERAL"))
 
   // Create an input parameter containing the path to this authentication file
-  def gcsAuthParameter(descriptor: WorkflowDescriptor) = {
-    if (AuthenticationMode == RefreshTokenMode || DockerHubCredentials.isDefined) {
+  def gcsAuthParameter(descriptor: WorkflowDescriptor): Option[JesInput] = {
+    if (JesConf.authMode == RefreshTokenMode || JesConf.dockerCredentials.isDefined) {
```

Comment:
I'll let you decide whether we care about the JesConf being half-mocked during tests. Otherwise :+1: